frost-core/src/tests/ciphersuite_generic.rs

```diff
@@ -196,8 +194,7 @@ pub fn check_sign<C: Ciphersuite + PartialEq, R: RngCore + CryptoRng>(
     key_packages: BTreeMap<frost::Identifier<C>, frost::keys::KeyPackage<C>>,
     mut rng: R,
     pubkey_package: PublicKeyPackage<C>,
-    signing_target: SigningTarget<C>,
-) -> Result<(SigningTarget<C>, Signature<C>, VerifyingKey<C>), Error<C>> {
```

Comment:
I added a test in `tweaking_tests.rs`, where I basically copied the README test and added tweaking. I don't like the repetition but I think it will do it for now.